src/service/url-replacements-impl.js

```diff
@@ -74,6 +74,22 @@ export class GlobalVariableSource extends VariableSource {
     /** @const {!./ampdoc-impl.AmpDoc} */
     this.ampdoc = ampdoc;
 
+    // A meta[name="amp-action-whitelist"] tag, if present, contains,
+    // in its content attribute, a whitelist of actions on the special AMP target.
+    if (this.ampVariableSubstitutionWhitelist_ === undefined
```

Comment:
This code is in a constructor, and there's no previous  assignment to `ampVariableSubstitutionWhitelist_`, so this condition should always be true.